data/transactions/logic/doc.go

```diff
@@ -219,84 +222,98 @@ func OpDoc(opName string) string {
-	"gtxnsa":  "{uint8 transaction field index} {uint8 transaction field array index}",
-	"txnas":   "{uint8 transaction field index}",
-	"gtxnas":  "{uint8 transaction group index} {uint8 transaction field index}",
-	"gtxnsas": "{uint8 transaction field index}",
-
-	"bnz":     "{int16 branch offset, big-endian}",
-	"bz":      "{int16 branch offset, big-endian}",
-	"b":       "{int16 branch offset, big-endian}",
-	"callsub": "{int16 branch offset, big-endian}",
-
-	"load":   "{uint8 position in scratch space to load from}",
-	"store":  "{uint8 position in scratch space to store to}",
-	"gload":  "{uint8 transaction group index} {uint8 position in scratch space to load from}",
-	"gloads": "{uint8 position in scratch space to load from}",
-	"gaid":   "{uint8 transaction group index}",
-
-	"substring": "{uint8 start position} {uint8 end position}",
-	"extract":   "{uint8 start position} {uint8 length}",
-	"replace2":  "{uint8 start position}",
-	"dig":       "{uint8 depth}",
-	"bury":      "{uint8 depth}",
-	"cover":     "{uint8 depth}",
-	"uncover":   "{uint8 depth}",
-
-	"asset_holding_get": "{uint8 asset holding field index}",
-	"asset_params_get":  "{uint8 asset params field index}",
-	"app_params_get":    "{uint8 app params field index}",
-	"acct_params_get":   "{uint8 account params field index}",
-
-	"itxn_field": "{uint8 transaction field index}",
-	"itxn":       "{uint8 transaction field index}",
-	"itxna":      "{uint8 transaction field index} {uint8 transaction field array index}",
-	"itxnas":     "{uint8 transaction field index}",
-	"gitxn":      "{uint8 transaction group index} {uint8 transaction field index}",
-	"gitxna":     "{uint8 transaction group index} {uint8 transaction field index} {uint8 transaction field array index}",
-	"gitxnas":    "{uint8 transaction group index} {uint8 transaction field index}",
-
-	"ecdsa_verify":        "{uint8 curve index}",
-	"ecdsa_pk_decompress": "{uint8 curve index}",
-	"ecdsa_pk_recover":    "{uint8 curve index}",
-
-	"base64_decode": "{uint8 encoding index}",
-	"json_ref":      "{uint8 return type index}",
-
-	"vrf_verify": "{uint8 parameters index}",
-	"block":      "{uint8 block field index}",
-
-	"switch": "{uint8 branch count} [{int16 branch offset, big-endian}, ...]",
-	"match":  "{uint8 branch count} [{int16 branch offset, big-endian}, ...]",
-
-	"proto":      "{uint8 arguments} {uint8 return values}",
-	"frame_dig":  "{int8 frame slot}",
-	"frame_bury": "{int8 frame slot}",
-	"popn":       "{uint8 stack depth}",
-	"dupn":       "{uint8 copy count}",
+var opcodeImmediateNotes = map[string][]string{
+	"intcblock":  {"const int block"},
+	"intc":       {"int constant index"},
+	"pushint":    {"int constant"},
+	"pushints":   {"int constants"},
+	"bytecblock": {"const byte block"},
+	"bytec":      {"byte constant index"},
+	"pushbytes":  {"byte constant"},
+	"pushbytess": {"byte constants"},
```

Comment:
What's going on?  These are all wrong, in the sense they don't convey what this used to be used to convey?